dragonflybsd/README.md

```diff
@@ -0,0 +1,137 @@
+# DragonflyBSD fwupd port notes
+
+## Preparation
+
+- Installed packages
+    - qemu-system-x86-64
+    - ovmf
+- Downloaded
+[latest version](https://mirror-master.dragonflybsd.org/iso-images/?C=M;O=D) of
+DragonflyBSD release in uncompressed `.iso` format
+- Created Virtual disk with `qemu-img` command:
+
+```
+$ qemu-img create -f qcow2 disk.qcow2 15G
+```
+
+> NOTE: HAMMER2 (default filesystem on DragonflyBSD) was designed to work with >
+50GB spaces. From 512MB to 1GB can be reserved for reblocking and UNDO/READ
+FIO. 15GB virtual disk was verified and should be enough for kernel development
+and verification purposes.
+
+## Scripts
+
+These scripts do not provide full automation but still can be useful
+
+- **vm/setup.sh** - run QEMU machine first time
+- **vm/run.sh** - run QEMU machine after instalation
+
+## Installation
+
+DragonflyBSD does not provide support for
+[autoinstall](https://man.openbsd.org/autoinstall) utility. We can found
+[non-official automation scripts](https://code.umbriel.fr/Nihl/dfly-autoinstall/src/branch/main/install-script/autoinstall.sh)
+, but it requires preparing custom `.ISO` with
+enabled output on `tty`, which is not available in the image downloaded from
+[DragonflyBSD webpage](https://www.dragonflybsd.org/download/#index1h2).
+
+To run DragonflyBSD first time, we need to create a virtual disk and run QEMU
+with
+mounted `.iso`:
+
+```
+$ qemu-img create -f qcow2 disk.qcow2 15G
+
+$ qemu-system-x86_64 -m 2048 -boot d \
+   -bios /usr/share/ovmf/x64/OVMF.fd \
+   -cdrom dfly-x86_64-6.2.2_REL.iso \
+   -drive if=virtio,file=disk.qcow2,format=qcow2 \
+   -netdev user,id=mynet0,hostfwd=tcp::7722-:22 \
+   -device e1000,netdev=mynet0 -enable-kvm -smp 4 -cpu host
+```
+
+The installation process isn't difficult; we need to answer a few questions (e.g.
+choose filesystem) and was described
+[here](https://www.dragonflybsd.org/docs/handbook/Installation/#index3h2).
+
+## Running DragonflyBSD in QEMU
+
+After installing DragonflyBSD we can run QEMU without mounted `.iso`:
+
```

Comment:
We could separate this into two subsections - manual and automatic (using vm/run.sh)